packages/webapp/src/pages/Integration/components/EnableDisableSync.tsx

```diff
@@ -88,17 +66,17 @@ export default function EnableDisableSync({
         setVisible(true);
     };
 
-    const createNewFlow = async (flow: ExtendedFlow) => {
+    const createNewFlow = async (body: PostPreBuiltDeploy['Body']) => {
         setModalShowSpinner(true);
         if (setIsEnabling) {
             setIsEnabling(true);
         }
-        const res = await createFlow([flow]);
+        const res = await createFlow(body);
 
-        return finalizeEnableSync(res, flow.model_schema);
+        return finalizeEnableSync(res);
     };
 
-    const reEnableFlow = async (flow: ExtendedFlow): Promise<boolean> => {
+    const reEnableFlow = async (flow: any): Promise<boolean> => {
```

Comment:
this `any` is temporary?

Reply:
yes indeed, I should have added a comment.